Feature: Point Priority Array

riteshakya037 opened the pull request.

RaiBnod committed:
Merge pull request #4 from NubeIO/release/1.1.0

Release/1.1.0

riteshakya037 committed:
Add clear point array

riteshakya037 committed:
Fix lint issues

RaiBnod commented:
Need to look on these:

1. Dashboard changes not reflected on network selection change (global variable)
2. Sync doesn't seem working well
3. Edit panel devices > points listing issue
4. Color encoding in writer priorities doesn't seems correct
5. Schedule selection change doesn't change the content

riteshakya037 committed:
Various Bug fixes

- Fix dashboard changes not reflected on network selection change (global variable)
- Fix Color encoding in writer priorities
- Fix issue with Multi switch not showing the right value

riteshakya037 committed:
Fix issue with series being null

riteshakya037 committed:
Fix state issues in some widgets

- Fix current selection in slider widget when refresh
- Fix clear in priority array writer
- Fix state issue in Multi Switch panel

RaiBnod commented:
Need to look at these:
- value re-calculation needs to be done as per mappings
- if the network doesn't exist from the global variable selection, it doesn't change the values and doesn't show the error either

<img width="1231" alt="image" src="https://user-images.githubusercontent.com/6800775/228716019-040aa564-a275-4bc4-8032-486da79b2ba4.png">

https://user-images.githubusercontent.com/6800775/228716045-d946b03e-7541-412f-913b-3961e09b4448.mov

riteshakya037 committed:
Add mapping value for Switch, MultiSwitch

riteshakya037 committed:
Disabled clear button if value is already null
Add write and default priority in UI

riteshakya037 committed:
Add info icon to show priority read/write conflict
